index.md

```diff
@@ -153,4 +153,4 @@ See the [Language support documentation](/docs/services/natural-language-underst
 ## Pricing
 {: #pricing}
 
-For pricing information, see the [Natural Language Understanding service](https://console.ng.bluemix.net/catalog/services/natural-language-understanding) in Bluemix.
+For pricing information, see the [Natural Language Understanding service](https://console.ng.bluemix.net/catalog/services/natural-language-understanding) in the {{site.data.keyword.cloud}} catalog.
```

Comment:
- need a conref for Natural Language Understanding (`{{site.data.keyword.nlushort}}`?)
- also, link shouldn't have the `ng.` so that it resolves to local region.
And I wonder whether the link for pricing should go to the dev console now (https://console.bluemix.net/developer/watson/services)